components/Cards/examples/EditReorderCollectionViewController.swift

```diff
@@ -27,7 +27,7 @@ class EditReorderCollectionViewController: UIViewController,
 
   let collectionView = UICollectionView(frame: .zero,
                                         collectionViewLayout: UICollectionViewFlowLayout())
-  var dataSource = [(Int, Bool)]()
+  var dataSource = [(Int, Bool, String)]()
```

Comment:
Looks like the `Bool` isn't used. Can we remove it?

Reply:
It is now used in new PR.